pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -14,9 +14,12 @@ style_min: true
   @header["title"] = hoc_s(:hoc2020_header)
 
 =inline_css 'homepage.css'
-=inline_css '/generated/hoc-banner.css'
--# TODO: Remove this when 'pl-launch-hero-banner' is set to 'false'
+-# Comment this out when we are out of HoC season
+-# =inline_css '/generated/hoc-banner.css'
```

Comment:
I commented this out and added the comment because these banner stylesheets are affecting each other.